WooCommerce/Classes/POS/Presentation/Barcode Scanner Setup/PointOfSaleBarcodeScannerSetupFlow.swift

```diff
@@ -326,9 +379,7 @@ private extension PointOfSaleBarcodeScannerSetupFlow {
     }
 
     private func trackRetry() {
-        if let step = getCurrentAnalyticsStepValue() {
-            analytics.track(event: WooAnalyticsEvent.PointOfSale.barcodeScannerSetupRetryTapped(scanner: scannerType))
-        }
```

Comment:
Just double checking, the `getCurrentAnalyticsStepValue` check isn't needed anymore?